### PR TITLE
breadth first

### PR DESCRIPTION
**Pull request by lhchandler4**
- [ X] Top-level README “Table of Contents” is updated
 - [ X] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [ X] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [ X] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code
     - [ X] Picture of whiteboard

### CODE REVIEW

File: challenges/src/main/java/Java/Tree/BinaryTree.java [hunk 1]
```diff
@@ -77,4 +81,43 @@ public void postOrderHelper(TreeNode<T> node, ArrayList<T> helperArrLi){
         }
         helperArrLi.add(node.data);
     }
+
+//    public void breadthFirst(BinaryTree input) {
```
Comment (evanslaton) [inline challenges/src/main/java/Java/Tree/BinaryTree.java:85]: Corpse code!

File: challenges/src/main/java/Java/Tree/BinaryTree.java [hunk 2]
```diff
@@ -77,4 +81,43 @@ public void postOrderHelper(TreeNode<T> node, ArrayList<T> helperArrLi){
...
+//        //dequeue
+//        while(q != null){
+//            System.out.println(q.remove());
+//            q.add(input.rootNode.leftChild);
+//            q.add(input.rootNode.rightChild);
+//        }
+//    }
+
+    static String breadthFirst(BinaryTree input) {
+        if(input.rootNode == null)
```
Comment (evanslaton) [inline challenges/src/main/java/Java/Tree/BinaryTree.java:98]: Not using brackets on a multi-line `if` statement is asking for trouble!

File: challenges/src/main/java/Java/Tree/BinaryTree.java [hunk 3]
```diff
@@ -77,4 +81,43 @@ public void postOrderHelper(TreeNode<T> node, ArrayList<T> helperArrLi){
...
+            return "null";
```
Comment (evanslaton) [inline challenges/src/main/java/Java/Tree/BinaryTree.java:99]: Why not just return `null` instead of the String null?
